Documentation/install/rhel/installing-workers.md

```diff
@@ -94,6 +94,12 @@ $ yum install tectonic-worker
 This will download the relevant dependencies and then prompt to validate the
 GPG key installed by the `tectonic-release` RPM.
 
+It is important to keep the Kubelet version in sync with the cluster's Tectonic version. To prevent incorrect `yum update` commands from updating the version, disable the Tectonic repo by writing `enabled=0` into:
```

Comment:
The Kubelet version must be kept in sync with the cluster's Tectonic version. ...